cmd/ocp.go

```diff
@@ -93,6 +93,17 @@ func openShiftCmd() *cobra.Command {
 		} else {
 			envVars["ALERTS"] = ""
 		}
+		esServerEnv := os.Getenv("ES_SERVER")
+		esIndexEnv := os.Getenv("ES_INDEX")
+		if workloadConfig.MetricsEndpoint == "" && (esServerEnv == "" || esIndexEnv == "") {
+			envVars["ES_SERVER"] = ""
+			envVars["ES_INDEX"] = ""
+		}
+		if esServer != "" && esIndex != "" {
+			envVars["ES_SERVER"] = esServer
+			envVars["ES_INDEX"] = esIndex
+		}
+		envVars["ES_INDEX"] = ""
```

Comment:
```suggestion
		envVars["ES_INDEX"] = ""
```
This specific line should be removed as it overrides and legit ES_INDEX value to empty string.